src/DIRAC/Resources/Computing/HTCondorCEComputingElement.py

```diff
@@ -173,6 +173,17 @@ def __writeSub(self, executable, nJobs, location, processors):
 
         executable = os.path.join(self.workingDirectory, executable)
 
+        useCredentials = ""
+        if tokenFile:
+            useCredentials = (
+                """
+use_x509userproxy = true
```

Comment:
does this need to stay "true"?

Reply:
I think yes. This is the way to tell condor to pass also proxy along with the token. If we choose always to bundle proxy in, this is not needed but does not harm neither